hawkular-rest-status/src/main/java/org/hawkular/commons/rest/status/ManifestUtil.java

```diff
@@ -54,6 +62,6 @@
                 }
             }
         }
-        return ret;
+        return Collections.unmodifiableMap(ret);
```

Comment:
OK, the unmodifiable wrapper does not need to be here. But I still vote for having it in `RestStatusHandler.getBaseStatus(ServletContext)` to make sure that the lazily initialized base map cannot be changed by any accident. https://github.com/ppalaga/hawkular-commons/commit/15a31442f4e6472be3d46db52b08f2ad6d85a213#diff-62f5e8b16e20d9e1db3c1994fc84f0ffR92